packages/terra-badge/src/_variables.scss

```diff
@@ -1,57 +1,80 @@
-$terra-badge-font-sizes: (
-  tiny: 0.5rem,
-  small: 0.8rem,
-  medium: 1rem,
-  large: 1.4rem,
-  huge: 1.7rem
-);
+:root {
```

Comment:
It was brought up that if someone is rolling their own theme (and not compiling custom props server side) that we may want an option to not include the default theme. Is it worth creating a Sass variable that default includes the theme but can be configured to leave it out for those that want to roll their own and not have duplicate variable definitions